internal/utils/file.go

```diff
@@ -23,10 +24,58 @@ func CreateFolder(path string) error {
 	return nil
```

Comment:
This is still using hardcoded `/vods/` (as well as a couple other places in this file).
I tried to archive a video from a new channel, it failed to create the folder (no permissions).
I've set `VIDEOS_DIR=/data/vods` and `TEMP_DIR=/data/temp`.
Furthermore, `entrypoint.sh` tries to chmod `/vods`, `/tmp`, etc.

Reply:
To clarify, the no permissions is because `/vods` does not exist and the abc user does not have write for `/`